src/pages/components/datepicker.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Datepicker
+description: >
+  A Datepicker lets users select a date by showing them a calendar. It works in combination with a text input.
+package: '@zendeskgarden/react-datepickers'
+components:
+  - datepickers/src/elements/Datepicker/Datepicker.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To select a single date</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>To select a range of dates, use DatepickerRange instead</Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Datepicker component.
+
+import Default from '../../examples/datepicker/datepicker/DatepickerDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/datepicker/datepicker/DatepickerDefault.tsx';
```

Comment:
Can we have the datepicker be the same width as the input? It's not quite the same size, which feels unresolved. 
![image](https://user-images.githubusercontent.com/1230973/93148638-30643180-f6a9-11ea-9dc4-7777e5c0b5f1.png)

Reply:
I would lean toward making the input larger actually. These shouldn't every align width unless it's a coincidence.

Reply:
I matched the Text input sizing from forms for now but don't feel strongly about either way.

Reply:
Gotcha. OK, then leaving as is is fine.